templates/openstackclient/bin/tripleo-export-ceph

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env python3
+
+import json
+import os
+import sys
+import tripleoclient.export
+import yaml
+
+CEPH_USER_FN = 'ceph_client_user.json'
+CEPH_USER_KEY = 'ceph_client_user'
+CEPH_DEPLOY_DIRS = (
+    'ceph-ansible',
+    'ceph-adm'
+)
+
+def get_cephx_name(playbooks_dir):
+    ceph_user_file_path = os.path.join(playbooks_dir, CEPH_USER_FN)
+    with open(ceph_user_file_path) as ceph_user_file:
+        data = json.loads(ceph_user_file.read())
+    return data.get(CEPH_USER_KEY, 'overcloud')
+
+def export_ceph_data(working_dir):
+    stackname = 'tripleo-ansible'
+    playbooks_dir = os.path.join(working_dir, 'playbooks')
+    output_file = os.path.join(playbooks_dir, 'ceph-export.yaml')
+
+    data = {}
+    data['parameter_defaults'] = {}
+
+    ceph_deploy_found = False
+    for ceph_deploy_dir in CEPH_DEPLOY_DIRS:
```

Comment:
Makes sense to me. Similar to:

  https://github.com/openstack/python-tripleoclient/blob/master/tripleoclient/v1/overcloud_export_ceph.py#L95

Reply:
ack, logic is based on that but I diverged slightly by setting merge strategies to make it easier to consume: user can just include multiple heat environment files and the entries will get appended to CephExternalMultiConfig